fuzzing/common.bzl

```diff
@@ -20,25 +20,28 @@ def _fuzzing_launcher_impl(ctx):
     script = ctx.actions.declare_file("%s" % ctx.label.name)
 
     script_template = """#!/bin/sh
-exec {launcher_path} {target_binary_path} "$@"
+exec {launcher_path} {target_binary_path} --corpus_dir={corpus_dir} "$@"
 """
 
     script_content = script_template.format(
         launcher_path = ctx.executable._launcher.short_path,
         target_binary_path = ctx.executable.target.short_path,
+        corpus_dir = ctx.file.corpus.short_path if ctx.attr.corpus else "",
```

Comment:
because this is "" if there's no corpus, won't the launcher script always have this flag?

Reply:
Yes, the flag will always be there, and in the launcher script, the default value of `corpus_dir` is also `""`(a empty string), which will be ignored later when passing the argument to the target binary to execute. Do you think I should add a judgement here to decide whether to pass the `corpus_dir`?

Reply:
I think handling it with a default empty string is nice, so that the launcher script can always except a corpus string (even if empty). I think my question is, won't the launcher script work fine even with an empty string for the corpus? 
In this case it should just run with no corpus, libfuzzer will still work without any corpus

Reply:
In my experiment, when appending `""` to `execv` argument list, it causes an error `No such file or directory: ; exiting`. Seems even passing an empty string, it's regarded as an directory path, which makes the `if` statement in the launcher script  necessary.

Reply:
Awesome, thank you for clarifying and testing that!